x/auth/tx/config/config.go

```diff
@@ -65,7 +69,10 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 		customSignModeHandlers = in.CustomSignModeHandlers()
 	}
 	sdkConfig := sdk.GetConfig()
+	// Append SIGN_MODE_TEXTUAL to the default sign modes.
+	enabledSignModes := append(tx.DefaultSignModes, signingtypes.SignMode_SIGN_MODE_TEXTUAL)
```

Comment:
does this need to be set in x/tx now?

Reply:
I don't think so, but I'm not sure what you mean

Reply:
the new x/tx module is handling all things related to signing, i thought things should be added there since auth/tx was depreacted?